src/Controller/SecurityController.php

```diff
@@ -34,6 +35,11 @@ class SecurityController extends AbstractController
      */
     public function login(Request $request, AuthenticationUtils $helper): Response
     {
+        // Block login page for logged in users
+        if ($this->container->get('security.authorization_checker')->isGranted('ROLE_USER')) {
+            return new RedirectResponse('/');
```

Comment:
To be consistent with other parts of the demo, use `return $this->redirectToRoute('blog_index');` instead of directly creating the RedirectResponse object.